config/routes.rb

```diff
@@ -1,3 +1,10 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  mount Avo::Engine, at: Avo.configuration.root_path
+  root 'cities#index'
+  resources :cities, only: [:index] do
+    post :forecast, on: :collection
```

Comment:
I feel like, these should be mapped on default routes instead of creating a custom route